### ISSUE TITLE
Image compression

### ISSUE THREAD

**Issue by eloopy** [HashLips/hashlips_art_engine#1119]
Hi everyone
Is there a way to apply compression to the layers?

**Comment by bolshoytoster** [comment]
@eloopy do you mean lowering the resolution or making a layer smaller?

**Comment by eloopy** [comment]
Yes. I've found a compression option on the canvas.toBuffer() command  inside saveimage() function.
But I was wondering if it's possible to apply this to every layer and also antialising?
Any help will be appriciated

**Comment by bolshoytoster** [comment]
@eloopy
> I've found a compression option ... But I was wondering if it's possible to apply this to every layer

that will be applied to the final image, so it will affect every layer.

If you want to apply it to each layer individually it's probably easiest to just compress your source images.
